plugins/linux/ifplugin/data_resync.go

```diff
@@ -50,13 +49,9 @@ type LinuxDataPair struct {
 func (plugin *LinuxInterfaceConfigurator) Resync(nbIfs []*interfaces.LinuxInterfaces_Interface) (errs []error) {
 	plugin.log.Debugf("RESYNC Linux interface begin.")
 
-	start := time.Now()
-	defer func() {
-		if plugin.stopwatch != nil {
-			timeLog := measure.GetTimeLog("linux-interface resync", plugin.stopwatch)
-			timeLog.LogTimeEntry(time.Since(start))
-		}
-	}()
+	defer func(t time.Time) {
+		plugin.stopwatch.TimeLog("set-interface-mac").LogTimeEntry(time.Since(t))
```

Comment:
Incorrect name for time log, there are several others, please check.

Reply:
you have found the last one